dev/validate_bom__config.py

```diff
@@ -821,6 +821,10 @@ def init_argument_parser(self, parser, defaults):
     add_parser_argument(
         parser, 'google_account_name', defaults, 'my-google-account',
         help='The name of the primary google account to configure.')
+    add_parser_argument(
```

Comment:
Can you make this default to None and only pass it through if it is set?
I think the default behavior for deploying a validation should be what customers see.

Reply:
Done